src/Arcus.Scripting.LogicApps/Scripts/Disable-AzLogicApp.ps1

```diff
@@ -0,0 +1,38 @@
+param
+(
+    [string][parameter(Mandatory = $false)]$SubscriptionId = "",
+    [string][parameter(Mandatory = $true)]$ResourceGroupName,
+    [string][parameter(Mandatory = $true)]$LogicAppName,
+    [string][parameter(Mandatory = $false)]$AccessToken = ""
+)
+
+try{
+    if($SubscriptionId -eq "" -or $AccessToken -eq ""){
+        # Request accessToken in case the script contains records
+        $token = Get-AzCachedAccessToken
+
+        $Global:acces_token = $token.AccessToken
+        $Global:subscriptionId = $token.SubscriptionId
+    }
+    else{
+        $Global:acces_token = $AccessToken
+        $Global:subscriptionId = $SubscriptionId
+    }
+    
+    Write-Host "Attempting to disable $LogicAppName"
+    $params = @{
+        Method = 'Post'
+        Headers = @{ 
+	        'authorization'="Bearer $Global:acces_token"
+        }
+        URI = "https://management.azure.com/subscriptions/$Global:subscriptionId/resourceGroups/$ResourceGroupName/providers/Microsoft.Logic/workflows/$LogicAppName/disable?api-version=2016-06-01"
```

Comment:
This should maybe be configurable? Using this as default or so?

Reply:
Do you mean assigning the default value $Global:SubscriptionId to the $subscriptionId-parameter?
When writing this, I'm thinking I shouldn't be assigning values to global-variables, as it might interfere with other operations.

Additionally, I'd like to move 'Get-AzCachedAccessToken.ps1' to the Arcus.Scripting.Security-module, but I'll have to rebase from master first.

Reply:
No, I mean the URI completly, because it only has the US Azure management variant, and not the other regions.

Reply:
Aha, I see what you mean.
The idea behind this is also to allow users to simply use this command to enable/disable a Logic App, as this currently isn't supported in the existing Az-functions.
So, perhaps it would be an idea to add a parameter that can be used to indicate which Azure Cloud is being referred to, where the public cloud is default.

For instance:
 - AzureEnvironmentName:
   - AzureCloud [default]
     -> https://management.azure.com/
   - AzureChinaCloud 
     -> https://management.chinacloudapi.cn/
   - AzureUSGovernment
     -> https://management.usgovcloudapi.net/
   - AzureGermanCloud
     -> https://management.microsoftazure.de/

This way, the caller doesn't have to bother building the actual URL themselves.

Reply:
Yes, exactly. But not blocking for this PR I would say. Can done with another issue another time. Was something I thought about when I saw the URL.

Reply:
Fixed/implemented in last commit.